sci_bert/models/pico_crf_tagger.py

```diff
@@ -0,0 +1,144 @@
+from typing import Dict, Optional, List, Any
+import logging
+import warnings
+
+from overrides import overrides
+import torch
+from torch.nn.modules.linear import Linear
+
+from allennlp.common.checks import check_dimensions_match, ConfigurationError
+from allennlp.data import Vocabulary
+from allennlp.models.model import Model
+from allennlp.modules import Seq2SeqEncoder, TimeDistributed, TextFieldEmbedder
+from allennlp.modules import ConditionalRandomField, FeedForward
+from allennlp.nn import InitializerApplicator, RegularizerApplicator
+import allennlp.nn.util as util
+from allennlp.training.metrics import F1Measure
+
+logger = logging.getLogger(__name__)
+
+@Model.register("pico_crf_tagger")
+class PicoCrfTagger(Model):
```

Comment:
How similar or different this is from the AllenNLP CrfTagger ?

Reply:
because data is annotated at `IO` and evaluation is necessarily token-level, the CRF stuff has modification.  Specifically, crf encoding & decoding options are restricted instead of allowing option to pass args.  Also turned off span-level metrics because not a real thing for PICO. the `get_metrics()` method also different because want to skip some of the classes (e.g. PICO doesnt care about performance on `NotPICO` class).